walkers/jsonschema-draft3.json

```diff
@@ -28,10 +28,5 @@
     "vocabulary": "http://json-schema.org/draft-03/schema#",
     "type": "object",
     "walker": "jsonschema-draft3"
-  },
-  "extends": {
```

Comment:
Hm, https://datatracker.ietf.org/doc/html/draft-zyp-json-schema-03#section-5.26 declares `extends` to a string, where as the JSON Schema test suite examples declare `extends` to one or many actual sub schemas only. Is the string representation in the spec wrong?